.github/workflows/build-pdf.yml

```diff
@@ -39,57 +40,57 @@ jobs:
 
     steps:
     # Checkout the repository
-    - name: Checkout repository
-      uses: actions/checkout@v3
-      with:
-        submodules: 'recursive'
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
 
     # Pull the latest RISC-V Docs container image
-    - name: Pull Container
-      run: docker pull riscvintl/riscv-docs-base-container-image:latest
+      - name: Pull Container
+        run: docker pull riscvintl/riscv-docs-base-container-image:latest
 
     # Build PDF and HTML.
-    - name: Build Files
-      run: make all
-      env:
-        VERSION: v${{ github.event.inputs.version }}
-        REVMARK: ${{ github.event.inputs.revision_mark }}
+      - name: Build Files
+        run: make all
+        env:
+          VERSION: v${{ github.event.inputs.version }}
+          REVMARK: ${{ github.event.inputs.revision_mark }}
```

Comment:
This part, which pre-dates the PR, make me a little sad because the CI build that ends in the artifacts has the following version string in the title page: `Version v, 2024-01-27:`. In other words, VERSION and REVMARK end up being empty here. Is it possible to fix this somehow?

Reply:
I'm sure we can get a default from the the makefile. I'll look into this today

Reply:
Since this is unrelated to this PR I've file https://github.com/riscv/riscv-cheri/issues/72 and plan to address this separately.